tests/CommonExtractorTest.php

```diff
@@ -953,6 +953,43 @@ public function testNoRetryOnCsvError(): void
         }
     }
 
+    public function testSshWithCompression(): void
+    {
+        $this->cleanOutputDirectory();
+        $config = $this->getConfig(self::DRIVER);
+        $config['parameters']['db']['ssh'] = [
+            'enabled' => true,
+            'keys' => [
+                '#private' => $this->getPrivateKey(self::DRIVER),
+                'public' => $this->getEnv(self::DRIVER, 'DB_SSH_KEY_PUBLIC'),
+            ],
+            'sshHost' => 'sshproxy',
+            'localPort' => '33056',
+            'compression' => true,
+        ];
+        $result = ($this->getApp($config))->run();
+        $this->assertExtractedData($this->dataDir . '/escaping.csv', $result['imported'][0]['outputTable']);
+        $this->assertExtractedData($this->dataDir . '/simple.csv', $result['imported'][1]['outputTable']);
+    }
+
+    public function testSshWithFalseCompressionConfigRow(): void
+    {
+        $this->cleanOutputDirectory();
+        $config = $this->getConfigRow(self::DRIVER);
+        $config['parameters']['db']['ssh'] = [
+            'enabled' => true,
+            'keys' => [
+                '#private' => $this->getPrivateKey(self::DRIVER),
+                'public' => $this->getEnv(self::DRIVER, 'DB_SSH_KEY_PUBLIC'),
+            ],
+            'sshHost' => 'sshproxy',
+            'localPort' => '33066',
+            'compression' => true,
+        ];
+        $result = ($this->getApp($config))->run();
+        $this->assertExtractedData($this->dataDir . '/simple.csv', $result['imported']['outputTable']);
```

Comment:
and the assertion is different with the same settings except local port that's highly unexpected.

Reply:
This is testing config row, the first one tests standard config  (2 different config definitions)

Reply:
and why it'S called 

``` 
testSshWithFalseCompressionConfigRow
-------------^
```

Reply:
omg, sorry:)  I thought I had removed that `False`, thanks!